pkg/utils/grub.go

```diff
@@ -160,6 +160,34 @@ func (g Grub) Install(target, rootDir, bootDir, grubConf, tty string, efi bool,
 			}
 		}
 
+		var foundFonts bool
+		for _, m := range []string{"ascii.pf2", "euro.pf2", "unicode.pf2"} {
+			err = fsutils.WalkDirFs(g.config.Fs, rootDir, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				}
+				if d.Name() == m && strings.Contains(path, g.config.Arch) {
+					fileWriteName := filepath.Join(bootDir, fmt.Sprintf("%s/%s-efi/fonts/%s", systemgrub, g.config.Arch, m))
+					g.config.Logger.Debugf("Copying %s to %s", path, fileWriteName)
+					fileContent, err := g.config.Fs.ReadFile(path)
+					if err != nil {
+						return fmt.Errorf("error reading %s: %s", path, err)
+					}
+					err = g.config.Fs.WriteFile(fileWriteName, fileContent, cnst.FilePerm)
+					if err != nil {
+						return fmt.Errorf("error writing %s: %s", fileWriteName, err)
+					}
+					foundFonts = true
+					return nil
+				}
+				return err
+			})
+			if !foundFonts {
```

Comment:
Shouldn't this be outside the for loop block?

Reply:
not really, I just missed adding the name here. We only want to warn the user about it if we cant find  font X so it needs to be under the loop.

Reply:
I just copied the code from the function above when we do the same for the grub modules. I cant really remember now why its done like this, but pretty sure it had its purpose....which I cant remember now XD

Reply:
Trying to remember here, maybe it was done that way because we needed all modules and any of those failing should fail, but in this case the fonts are not really needed, its just nice to haver, so maybe the approach has to be different. No need for this variable anymore, can just warn inside the loop and call it a day....